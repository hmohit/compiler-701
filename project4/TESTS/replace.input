This is input to test replace.
See if all 3-X sequences are replaced by a 3-Y sequence.
example: XXX here
XX not enough Xs
XXX at start of line
at end of line: XXX
wrapping? XX
X
What about 3-Ys: YYY
No whitespace: aXXX XXXb aXXXb
Other chars between: aXbXcXd
