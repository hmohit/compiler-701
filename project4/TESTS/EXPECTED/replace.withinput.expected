This is input to test replace.
See if all 3-X sequences are replaced by a 3-Y sequence.
example: YYY here
XX not enough Xs
YYY at start of line
at end of line: YYY
wrapping? XX
X
What about 3-Ys: YYY
No whitespace: aYYY YYYb aYYYb
Other chars between: aXbXcXd
